java-checks/src/main/java/org/sonar/java/checks/EmptyMethodsCheck.java

```diff
@@ -57,30 +59,27 @@ public void visitNode(Tree tree) {
     ClassTree classTree = (ClassTree) tree;
     if (!ModifiersUtils.hasModifier(classTree.modifiers(), Modifier.ABSTRACT)) {
       for (Tree member : classTree.members()) {
-        if (member.is(Kind.METHOD)) {
+        if (member.is(Kind.METHOD) || isPublicConstructor(member)) {
           checkMethod((MethodTree) member);
         }
       }
     }
   }
 
+  private static boolean isPublicConstructor(Tree node) {
+    return node.is(Kind.CONSTRUCTOR) && ModifiersUtils.hasModifier(((MethodTree) node).modifiers(), Modifier.PUBLIC);
```

Comment:
The jira ticket is about no-arg constructors but here you only look at public constructors without looking at arguments. Is it expected?